src/custom/components/Settings/index.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useContext, useRef, useState } from 'react'
+import { Settings } from 'react-feather'
+import { Text } from 'rebass'
+import styled, { ThemeContext } from 'styled-components'
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { ApplicationModal } from 'state/application/actions'
+import { useModalOpen, useToggleSettingsMenu } from 'state/application/hooks'
+import {
+  useExpertModeManager,
+  useUserTransactionTTL,
+  useUserSlippageTolerance,
+  useUserSingleHopOnly
+} from 'state/user/hooks'
+import { TYPE } from 'theme'
+import { ButtonError } from 'components/Button'
+import { AutoColumn } from 'components/Column'
+import Modal from 'components/Modal'
+import QuestionHelper from 'components/QuestionHelper'
+import { RowBetween, RowFixed } from 'components/Row'
+import Toggle from 'components/Toggle'
+import TransactionSettings from 'components/TransactionSettings'
+import { StyledCloseIcon, StyledMenu, MenuFlyout, Break, ModalContentWrapper } from './SettingsMod'
```

Comment:
Continuing the suggestion:

```suggestion
import { StyledCloseIcon, StyledMenu, MenuFlyout, Break, ModalContentWrapper, SettingsTab as SettingsTabMod } from './SettingsMod'
```